### PR TITLE
Make tooltips set by using `.SetToolTip()` in designers translatable

### LINKED ISSUE #10717: Untranslatable tooltips in Pull dialog
### Environment

- Git Extensions 4.0.2.16100
- Build 25100ec1f7c8da8f798613da74826038af81a50e
- Git 2.38.1.windows.1
- Microsoft Windows NT 10.0.22621.0
- .NET 6.0.13
- DPI 96dpi (no scaling)
- Microsoft.WindowsDesktop.App Versions

```
    Microsoft.WindowsDesktop.App 5.0.5 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]
    Microsoft.WindowsDesktop.App 6.0.13 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]
```

### Issue description

There are untranslatable **tooltips** in the Pull dialog.
image

### Steps to reproduce

Open the Pull dialog.

### Did this work in previous version of GitExtensions?

_No response_

### Diagnostics

_No response_

### PR DESCRIPTION
**Pull request by pmiossec**
Fixes #10717

Alternative to #10824 (Thanks for their efforts!)

### After

image

## Test methodology 

- Manual

## Test environment(s) 

- Git Extensions 33.33.33
- Build 1a352606f72a94fba1914d8364554a27ccd47816 (Dirty)
- Git 2.38.1.windows.1
- Microsoft Windows NT 10.0.19045.0
- .NET 6.0.14
- DPI 96dpi (no scaling)

## Merge strategy

I agree that the maintainer squash merge this PR (if the commit message is clear).

----

:black_nib: I contribute this code under [The Developer Certificate of Origin](../blob/master/contributors.txt).

### CODE REVIEW

**Comment by pmiossec** [comment]
Note: the french translation file is for those that want to test but I will remove it before merging (if it will happen one day)

**Review by gerhardol — COMMENTED** [review]
+1

**Review by mstv — APPROVED** [review]
LGTM

File: ResourceManager/Xliff/TranslationUtil.cs [hunk 1]
```diff
@@ -190,6 +207,26 @@ public static void TranslateItemsFromList(string category, ITranslation translat
                     continue;
                 }
 
+                if (itemObj is ToolTip tooltip)
+                {
+                    string ProvideDefaultValue() => "";
```
Comment (RussKie) [inline ResourceManager/Xliff/TranslationUtil.cs:212]:
```suggestion
                    static string ProvideDefaultValue() => "";
```

Comment (RussKie) [reply ResourceManager/Xliff/TranslationUtil.cs:212]: Why not `=> null`?

**Review by RussKie — COMMENTED** [review]
👍

File: ResourceManager/Xliff/TranslationUtil.cs (hunk 1 above)
Comment (pmiossec) [reply ResourceManager/Xliff/TranslationUtil.cs]: I have hesitated to change it when copy/pasting from previous definition. So no real reason except the copy/paste. So changed...

**Comment by pmiossec** [comment]
I have made the changes. It seems good. Merge at will...

**Pull request merged**